constants/prometheus.go

```diff
@@ -25,8 +25,8 @@ const (
 	FeeAccountAmountGaugeHelp               = "Fee account amount."
 	BridgeAccountAmountGaugeName            = "bridge_account_amount"
 	BridgeAccountAmountGaugeHelp            = "Bridge account amount."
-	OperatorAccountAmountName               = "operator_account_amount"
-	OperatorAccountAmountHelp               = "Operator account amount."
+	AccountBalanceGaugeNamePrefix           = "account_balance_for_"
+	AccountBalanceGaugeHelpPrefix           = "Account balance for "
```

Comment:
Looks like this one is not used, maybe it should be used at https://github.com/LimeChain/hashport-validator/blob/feature/balances-for-each-member/app/process/watcher/prometheus/watcher.go#L123

Reply:
Ops yep I used the Name on the place where the Help needs to be :D